README.md

```diff
@@ -184,197 +184,202 @@ mdsf init
+| `gofumpt`              | `gofumpt -w PATH`                                                                      |
+| `goimports-reviser`    | `goimports-reviser -format PATH`                                                       |
+| `goimports`            | `goimports -w PATH`                                                                    |
+| `golines`              | `golines -w PATH`                                                                      |
+| `google-java-format`   | `google-java-format -i PATH`                                                           |
+| `grain:format`         | `grain format PATH -o PATH`                                                            |
+| `haml-lint`            | `haml-lint --auto-correct PATH`                                                        |
+| `hfmt`                 | `hfmt -w PATH`                                                                         |
+| `hindent`              | `hindent PATH`                                                                         |
+| `html-beautify`        | `html-beautify -r --type html -f PATH`                                                 |
+| `htmlbeautifier`       | `htmlbeautifier PATH`                                                                  |
+| `imba:fmt`             | `imba fmt -f PATH`                                                                     |
+| `isort`                | `isort --quiet PATH`                                                                   |
+| `joker`                | `joker --format --write PATH`                                                          |
+| `js-beautify`          | `js-beautify -r --type js -f PATH`                                                     |
+| `jsona:format`         | `jsona format PATH`                                                                    |
+| `jsona:lint`           | `jsona lint PATH`                                                                      |
+| `jsonnetfmt`           | `jsonnetfmt -i PATH`                                                                   |
+| `juliaformatter.jl`    | `julia -E using JuliaFormatter;format_file(\"\")`                                      |
+| `just`                 | `just --fmt --unstable --justfile PATH`                                                |
+| `kcl:fmt`              | `kcl fmt PATH`                                                                         |
+| `kdlfmt`               | `kdlfmt format PATH`                                                                   |
+| `ktfmt`                | `ktfmt --format --log-level=error PATH`                                                |
+| `ktlint`               | `ktlint --format --log-level=error PATH`                                               |
+| `kulala-fmt`           | `kulala-fmt PATH`                                                                      |
```

Comment:
Could you update the template to include the list of format it support, like a third column, here it would `http`